apiserver/facades/agent/uniter/uniter.go

```diff
@@ -41,6 +41,7 @@ import (
 	"github.com/juju/juju/domain/unitstate"
 	"github.com/juju/juju/internal/charm"
 	internalerrors "github.com/juju/juju/internal/errors"
+	internalrelation "github.com/juju/juju/internal/relation"
```

Comment:
issue: We don't need a 3rd relation package in this file when the `Endpoint` is the same in `domain/relation` and `internal/relation`.

Reply:
Have removed.